src/globus_sdk/services/flows/client.py

```diff
@@ -587,14 +587,14 @@ def get_run(
                     :ref: Flows/paths/~1runs~1{run_id}/get
         """
 
-        additional_query_params = query_params or {}
+        consolidated_query_params = query_params or {}
+        if include_flow_description is not None:
+            value = str(include_flow_description).lower()  # "true" or "false"
+            consolidated_query_params["include_flow_description"] = value
 
         return self.get(
             f"/runs/{run_id}",
-            query_params={
-                "include_flow_description": include_flow_description,
-                **additional_query_params,
-            },
+            query_params=consolidated_query_params,
         )
```

Comment:
Is there a reason not to simply call it `query_params`?
i.e.
```python
if query_params is None:
    query_params = {}
```

(or inline that if line count is the enemy :wink: )

Reply:
> is fixing this client-side the correct approach

Yes, because `requests` does not produce the documented values, and `responses` won't match unless the capitalization is correct.